data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSink.java

```diff
@@ -498,31 +495,17 @@ SerializedJson getDocument(final Event event) {
     }
 
     String routingValue = null;
-    if (routingField != null) {
```

Comment:
Why was this removed?

Reply:
wasn't sure if routingField was still relevant since it isn't listed in the documentation anymore

Reply:
Routing is still in use.